### DISCUSSION TITLE
可行性讨论：动态文字替换并粒子化特效

### DISCUSSION THREAD

**Discussion by n3g2** [Tencent/libpag#2581] [General] [answered]
文本粒子效果示例：
![4853a6d89011d758b045a38d5db13cfd](https://github.com/user-attachments/assets/856739cb-24b7-4f93-beff-d1488f4b47fe)

**Comment by kevingpqi123** [comment, marked as answer]
AE 中的特效包括第三方特效插件很多，这些都是需要通过开发相关功能才能实现矢量导出，进而进行文本编辑和占位图替换，如果不需要编辑，可以通过 BMP 预合成导出
